### PR TITLE
[EDU-3708] feat: add guide Waiting Room

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-3708] 

### Changes

- Add a how-to guide for Waiting Room integration (EN&PT)
- Add links to the guide on the Guides Page.

### Additional links

n/a.

[EDU-3708]: https://aziontech.atlassian.net/browse/EDU-3708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/guides/marketplace/integrations/waiting-room.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+2. Click the **New Rule** button.
+3. Give an easy-to-remember name to the rule.
+4. Select a *criteria* to run and catch the domains that you want to run the integration on. Example: if `Request URI` *starts with* `./`.
+5. Below, select the **Run Function** *behavior* to the *criteria*.
+6. Select the **Upstash Waiting Room** function according to the name you gave it before.
+7. Click the **Save** button.
+
+Done. Now the **Upstash Waiting Room** integration is running.
+
+
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/waiting-room.mdx:108]:
```suggestion
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+---
+
+## Obtenha a integração
+
+Para instalar esta integração:
+
+1. Acesse [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Marketplace**.
+2. Na página inicial do Marketplace, selecione o cartão da **Upstash Waiting Room**.
+3. Na página de integração, clique no botão **Get It Now** , localizado no canto inferior direito da página.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx:30]:
```suggestion
3. Na página de integração, clique no botão **Get It Now**, localizado no canto inferior direito da página.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+3. Dê um nome fácil de lembrar à sua nova regra.
+4. Selecione os domínios que deseja proteger com a função.
+5. Ative a opção **Edge Functions** na seção **Edge Firewall Modules**.
+- Esta ação dá acesso a edge functions no edge firewall.
+6. Clique no botão **Save**.
+
+Pronto. Agora você instanciou a regra para sua função e tem acesso a edge functions em seu edge firewall.
+
+:::caution[Atenção]
+Se um produto ou módulo estiver ativado, pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/en/pricing/) para obter mais informações.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx:59]:
```suggestion
Se um produto ou módulo estiver ativado, pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para obter mais informações.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  "capacity": 2
+}
+```
+
+Onde:
+
+| Variável | Obrigatória | Descrição |
+| ---- | ---- | ---- |
+| `url`  | Sim | String de URL para conectar no Upstash Redis. Acesse o console do Upstash e copie o `UPSTASH_REDIS_REST_URL` na página do banco de dados |
+| `token` | Sim | Token para acessar o banco de dados Upstash Redis. Acesse o console do Upstash e copie o `UPSTASH_REDIS_REST_TOKEN na página` do banco de dados |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx:88]:
```suggestion
| `token` | Sim | Token para acessar o banco de dados Upstash Redis. Acesse o console do Upstash e copie o `UPSTASH_REDIS_REST_TOKEN` na página do banco de dados |
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+| `ttl` | Sim | Tempo máximo, em segundos, um usuário pode permanecer no seu site. Exemplo: `180` |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx:89]:
```suggestion
| `ttl` | Sim | Tempo máximo, em segundos, que um usuário pode permanecer no seu site. Exemplo: `180` |
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+### Configure uma regra no Rules Engine
+
+Para concluir, você deve configurar uma regra no **Rules Engine** para configurar os critérios (*criteria*) e o comportamento (*behavior*) para executar a função.
+
+Ainda na página do **Edge Firewall**:
+
+1. Selecione a aba **Rules Engine**.
+2. Clique no botão **New Rule**.
+3. Dê um nome fácil de lembrar à sua nova regra.
+4. Selecione um *criteria* (critério) para executar a solução
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx:101]:
```suggestion
4. Selecione um *criteria* (critério) para executar a solução.
```

**Review by hannah-sk — APPROVED** [review]
deixei alguns ajustes

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+### Configure uma regra no Rules Engine
+
+Para concluir, você deve configurar uma regra no **Rules Engine** para configurar os critérios (*criteria*) e o comportamento (*behavior*) para executar a função.
+
+Ainda na página do **Edge Firewall**:
+
+1. Selecione a aba **Rules Engine**.
+2. Clique no botão **New Rule**.
+3. Dê um nome fácil de lembrar à sua nova regra.
+4. Selecione um *criteria* (critério) para executar a solução.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/waiting-room.mdx:101]:
```suggestion
4. Selecione um *criteria* (critério) para executar a integração.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/waiting-room.mdx [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+### Setting up the Rules Engine
+
+To finish, you must set up a rule in **Rules Engine** to configure the **criteria** and the **behavior** to run the function.
+
+Still in the **Edge Firewall** page:
+
+1. Select the **Rules Engine** tab.
+2. Click the **New Rule** button.
+3. Give an easy-to-remember name to the rule.
+4. Select a *criteria* to run and catch the domains that you want to run the integration on. Example: if `Request URI` *starts with* `./`.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/waiting-room.mdx:101]:
```suggestion
4. Select a *criteria* to run and catch the domains that you want to run the integration on.
- Example: if `Request URI` *starts with* `./`.
```

**Pull request merged**